pwsh_module/pwsh_bolt.psm1.erb

```diff
@@ -90,3 +90,20 @@ end
 
 }
 <% end %>
+
+<% unless ENV['BOLT_DISABLE_PWSH_WARNING'] -%>
+function Write-DeprecationWarning
+{
+  $FirstRunFile = "$PSScriptRoot/Private/first_run"
```

Comment:
You're going to have to put this in the user's bolt directory (`~/.puppetlabs/bolt`). You won't be able to write to the `Program Files\WindowsPowerShell\Modules` directory on a production system.